test/utils.js

```diff
@@ -172,6 +173,19 @@ export const readFee = (amount, feeStr) => {
   }
 };
 
+export const separator = length => {
+  const terminalCols =
+    process.platform === 'win32'
+      ? length || 40
+      : length || parseInt(execSync(`tput cols`).toString());
```

Comment:
Is this `separator` function used somewhere?

A user recently had a problem with `tput` not installed on an alpine linux (see https://github.com/opencollective/opencollective/issues/1443). Maybe we should encapsulate it in a separate `getTerminalCols` which would returns the default `40` if the call fails.

Reply:
Could happen on Windows machine (but this is tested here) or on Alpine linux if we ever run our tests in a such container (for example in CI). We could just put that in a `try..catch` block and return `40` if we catch.

Reply:
Good idea. Done in this commit: https://github.com/opencollective/opencollective-api/pull/1448/commits/329526f32e1fee46de7081c103572ea06052065f